telegram_integration.py

```diff
@@ -49,6 +49,19 @@ def newgame_handler(bot, update, chat_data):
         chat_data["game_obj"] = Secret_Hitler.Game(chat_id)
         bot.send_message(chat_id=chat_id, text="Created game! /joingame to join, /startgame to start")
 
+def cancelgame_handler(bot, update, chat_data):
+    """
+    Cancel a game.
+    """
+    game = chat_data.get("game_obj")
+
+    chat_id = update.message.chat.id
+    if game is not None:
+        game.set_game_state(Secret_Hitler.GameStates.GAME_OVER)
+        bot.send_message(chat_id=chat_id, text="Game cancelled. /newgame to start a new one.")
```

Comment:
```suggestion
        raise GameOverException("Game cancelled. Type /newgame to start a new one.")
```

Reply:
Are you sure? I copied this from just above in `newgame_handler` where my variant is used to end the game.

Reply:
Oh wait, I confused the line this change was on, sorry.